content/rs/administering/database-operations/migrate-to-active-active.md

```diff
@@ -0,0 +1,96 @@
+{{< /note >}}
+
+To migrate an RS database to Active-Active:
+
+1. Create a new [Active-Active database]({{< relref "/rs/administering/creating-databases/create-active-active.md" >}}).
+
+    After the Active-Active database is activated, you see the database's configuration.
+
+1. Click **Edit** at the bottom of the database configuration.
+1. Enable **Migration using Replica Of**.
+1. Click **Continue** to confirm that you want to flush the data from the Active-Active database.
+1. Enter the URL of the source database endpoint:
+
+    <!-- Shared in create-active-passive.md -->
+
+    {{< note >}}
+The order of the Replica Of sources has no impact on replication.
+    {{< /note >}}
+
+    - For a source database in the same RS cluster - When you click on the box,
+    the available databases are shown in the correct format for the URL of the source endpoint:
+
+        ```sh
+        <database name>: redis://admin:<database_password>@<database_endpoint>:<database_port>
+        ```
+
+        You can select the database that you want to use as the source.
+
+    - For a source database in a different RS cluster:
+        1. Log in to the Web UI of the cluster that hosts the source database.
+        1. In **databases**, click on the database and go to **configuration**.
+        1. Under **Endpoint**, click on **Get Replica Of source URL**.
+
+            ![Replica Of source URL](/images/rs/replicaof-source-url.png)
+
+        1. Click **Copy to Clipboard** to copy the URL of the source endpoint.
+
+            If you want a different internal password, you can click **Regenerate Password**.
+
+            {{< warning >}}
+If you regenerate the password, replication to existing destinations fails until you update their configuration with the new password.
+            {{< /warning >}}
+
+        1. In the destination database, paste the URL of the source endpoint in the **Replica Of** box, and click ![Save](/images/rs/icon_save.png#no-click "Save").
+
+        {{< note >}}
+For a source database on a different RS cluster,
+you can [compress the replication data]({{< relref "/rs/administering/designing-production/active-passive#data-compression-for-replica-of">}}) to save bandwidth.
+        {{< /note >}}
+
+    - For a source database in an OSS Redis cluster - Enter the URL of the source endpoint in the format:
+
+        - If the database has a password -
+
+            ```sh
+            redis://:<redis_password>@<hostname>:<database_port>
+            ```
+
+            Where the password is the Redis password represented with URL encoding escape characters.
+
+        - If the database has no password -
+
+            ```sh
+            redis://<hostname>:<database_port>
+            ```
+
+    {{< note >}}
+If you used the mDNS protocol for the cluster name (FQDN),
+make sure that the [client mDNS prerequisites]({{< relref "/rs/installing-upgrading/configuring/mdns.md" >}}) are met.
```

Comment:
```suggestion
the [client mDNS prerequisites]({{< relref "/rs/installing-upgrading/configuring/mdns.md" >}}) must be met in order to communicate with other clusters.
```
@AlonMagrafta I want to make clear the consequence of not meeting the prerequisites. Do you think this is clear enough for the user?